app/models/project.rb

```diff
@@ -1,3 +1,4 @@
 class Project < ApplicationRecord
-  belongs_to :manager
+  belongs_to :manager, class_name: 'User'
+  has_many :bugs
 end
```

Comment:
Associations missing for the requirement "Manager can assign multiple developers and QAs to a project"